### ISSUE TITLE
Warn about missing `width` option

### ISSUE THREAD

**Issue by jazen** [borgfriend/gatsby-remark-embed-video#3]
I accidentally deleted the `options` from my configuration: 

```
{
  resolve: `gatsby-remark-embed-video`,
  options: {
  }
}
```

And instead of warning me about it the transformed `<iframe>` tags had a `width` attribute of `undefined` and `height` of `NaN`.

**Comment by borgfriend** [comment]
It seems that there is an issue where the default parameter is not read. I'll fix that in the next days. 
Thank you for opening the issue

**Comment by borgfriend** [comment]
Fixed in Version 1.1.0
